projects/pages/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@skyux/pages",
+  "version": "5.0.0-beta.3",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@skyux/i18n": "^5.0.0-beta.0",
+    "@skyux/indicators": "^5.0.0-beta.0",
+    "@skyux/layout": "^5.0.0-beta.0",
+    "@skyux/router": "^5.0.0-beta.0",
+    "@skyux/theme": "^5.0.0-beta.0"
```

Comment:
These versions should match the version listed in the workspace package.json, e.g., "@skyux/i18n": "^5.0.0-beta.7".